src/components/appheader.main.jsx

```diff
@@ -103,6 +105,7 @@ class AppHeaderMain extends React.Component {
     const {
       isOffline, cartData, isLoading, isSearchFocused,
     } = this.state;
+    const availability = !!cartData;
```

Comment:
cartData should not control if other elements on the page are viewable. cartData will only control if the cart can be reached. There are multiple resources for various other features. We should rename this variable to `cartAvailability` to not cause confusion
example: defaultProfile will determine if the profile data is available.